docs/how-to-work-on-guide-topics.md

```diff
@@ -1,57 +1,69 @@
 
-Here are some examples of properly named titles:
+### When do we edit or remove topics
 
-> [`guide/english/javascript/loops/for-loop/index.md`](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/javascript/loops/for-loop/index.md)
+We may change or remove a topic
 
-```markdown
----
-title: For Loop
----
-```
+- if there is not a relevant title used in the topic title
+- if there is zero/little effort in it (e.g: copy-pasting from another source like Wikipedia)
+- if there is copied text from a copyrighted source
+- if it does not respect the [Style guide for writing topics](#style-guide-for-creating-and-editing-guide-topics)
+- if it does not respect the [Academic Honesty policy](https://www.freecodecamp.org/academic-honesty)
+- if any of the content violates the [freeCodeCamp Code of Conduct](https://www.freecodecamp.org/news/code-of-conduct)
+- if a new Guide topic wiki only contains links and not other relevant content
 
-> [`guide/spanish/algorithms/binary-search-trees/index.md`](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/spanish/algorithms/binary-search-trees/index.md)
+### Getting Help
 
-```
----
-title: Binary Search Trees
-localeTitle: Árboles binarios de búsqueda
----
-```
+There's a community of support from a whole team of contributors, whom you can bounce ideas off of and ask for input on your writing.
+
+Stay active in the [contributors' chat room](https://gitter.im/freecodecamp/contributors) and ask lots of questions.
+
+### Style guide for creating and editing Guide topics
 
-## Modularity
+We recommend the following guidelines for writing Guide topics to help you get started contributing and to create helpful topics.
 
-Each article should explain exactly one concept, and that concept should be apparent from the article's title.
+#### Title
 
-We can reference other articles by linking to them inline, or in an "Other Resources" section at the end of the article.
+topic titles should be as short, concise, and as to-the-point as possible.
 
-Our goal is to have thousands of articles that cover a broad range of technical topics.
+We want campers to quickly find the information they're looking for, and the title should reflect the main theme of the topic.
 
-## Code Blocks
+#### Modularity
 
-Campers will likely use Guide articles as a quick reference to look up syntax. Articles should have simple real-world examples that show common-use cases of that syntax.
+Each topic should explain exactly one concept, and that concept should be apparent from the topic's title.
 
-GitHub-flavored markdown supports [syntax highlighting in code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting) for many programming languages.
+We can reference other topics by linking to them inline, or in an "Other Resources" section at the end of the topic.
+
+Our goal is to have thousands of topics that cover a broad range of technical topics.
```

Comment:
Possible redundant use of "topics", maybe something like this would work? 
```suggestion
Our goal is to have thousands of topics that cover a broad range of technical concepts.
```